cookbooks/ros2_windows/templates/qt-maintenance.qs.erb

```diff
@@ -152,7 +152,10 @@ Controller.prototype.WelcomePageCallback = function WelcomePageCallback() {
 };
 
 Controller.prototype.CredentialsPageCallback = function CredentialsPageCallback() {
-  gui.clickButton(buttons.CommitButton);
+  var page = gui.pageWidgetByObjectName('CredentialsPage');
```

Comment:
Is this necessary on the maintenance one? I'd assume the credentials would already be saved somewhere...

Reply:
This covers the case that you're maintaining an existing install whose credentials have expired. Since this callback is hopefully only fired when the credentials page is shown it hopefully won't have deleterious effects otherwise.